tools/rfc-render/render-rfc-table.js

```diff
@@ -3,11 +3,14 @@ const fs = require('fs').promises;
 const path = require('path');
 
 const labels = {
-  'status/ready': { },
+  'status/implementing': { },
+  'status/planning': { },
+  'status/approved': { },
   'status/final-comment-period': { }, 
-  'status/pending': { },
+  'status/review': { },
   'status/proposed': { },
-  'status/resolved': { },
+  'status/done': { },
+  'status/rejected': { },
```

Comment:
Nitpick: Does ordering matter here? Logically, I'd either arrange by alpha (order doesn't matter) or chronologically (proposed, review, fcp...).

Reply:
Yes, ordering matters and intentional, sorted by when it will be delivered (chronological in reverse order). The top items are the ones actually being worked on, then the ones in planning, approved, etc. I put the "done" items in the end because they are less interesting in this list.

Reply:
I'll add a comment